addons/skin.confluence/720p/VideoOSD.xml

```diff
@@ -248,9 +253,8 @@
 				<onup>1000</onup>
 				<ondown>1000</ondown>
 				<onclick>PlayerControl(ShowVideoMenu)</onclick>
-				<enable>VideoPlayer.HasMenu</enable>
 				<animation effect="fade" start="100" end="50" time="75" condition="!VideoPlayer.HasMenu">Conditional</animation>
```

Comment:
This fade animation can go now to, since its not visible anymore when the fade kicks in.